projects/models.py

```diff
@@ -0,0 +1,281 @@
+        _('Topic'),
+        blank=True,
+        null=True,
+        max_length=MAX_TITLE_LENGTH,
+    )
+    duration = models.DurationField(  # timedelta
+        _('Duration'),
+        blank=False,
+        null=False,
+    )
+
+    begin_date = models.DateTimeField(  # datetime
+        _('Start date'),
+        blank=True,
+        null=True,
+    )
+    end_date = models.DateTimeField(  # datetime
+        _('End date'),
+        blank=True,
+        null=True,
+    )
+
+    external_venue = models.BooleanField(
+        _('External venue'),
+        blank=False,
+        null=False,
+    )
+
+    jugendmedienschutz = models.BooleanField(
+        _('Jugendmedienschutz'),
+        blank=False,
+        null=False,
+    )
+
+    target_group = models.ForeignKey(
+        TargetGroup,
+        on_delete=models.CASCADE,
+        default=default_target_group
+    )
+
+    project_category = models.ForeignKey(
+        ProjectCategory,
+        on_delete=models.CASCADE,
+        default=default_category
+    )
+
+    project_leader = models.ForeignKey(
+        ProjectLeader,
+        on_delete=models.CASCADE,
+        blank=False,
+        null=False,
+    )
+
+    media_education_supervisor = models.ForeignKey(
+        MediaEducationSupervisor,
+        on_delete=models.CASCADE,
+        default=default_category
+    )
+
+    tn_0_bis_6 = models.IntegerField(
+        _('bis 6 Jahre'),
+        blank=False,
+        null=False,
+        default=0
+    )
+    tn_7_bis_10 = models.IntegerField(
+        _('7-10 Jahre'),
+        blank=False,
+        null=False,
+        default=0
+    )
+    tn_11_bis_14 = models.IntegerField(
+        _('11-14 Jahre'),
+        blank=False,
+        null=False,
+        default=0
+    )
+    tn_15_bis_18 = models.IntegerField(
+        _('15-18 Jahre'),
+        blank=False,
+        null=False,
+        default=0
+    )
+    tn_19_bis_34 = models.IntegerField(
+        _('19-34 Jahre'),
+        blank=False,
+        null=False,
+        default=0
+    )
+    tn_35_bis_50 = models.IntegerField(
+        _('35-50 Jahre'),
+        blank=False,
+        null=False,
+        default=0
+    )
+    tn_51_bis_65 = models.IntegerField(
+        _('51-65 Jahre'),
+        blank=False,
+        null=False,
+        default=0
+    )
+    tn_ueber_65 = models.IntegerField(
+        _('über 65 Jahre'),
+        blank=False,
+        null=False,
+        default=0
+    )
+    tn_age_not_given = models.IntegerField(
+        _('ohne Angabe'),
+        blank=False,
+        null=False,
+        default=0
+    )
+
+    tn_female = models.IntegerField(
+        _('weiblich'),
+        blank=False,
+        null=False,
+        default=0
+    )
+    tn_male = models.IntegerField(
+        _('männlich'),
+        blank=False,
+        null=False,
+        default=0
+    )
+    tn_gender_not_given = models.IntegerField(
+        _('ohne Angabe'),
+        blank=False,
+        null=False,
+        default=0
+    )
+
```

Comment:
In our profiles we also have the gender `diverse`. I don't know if we should include it here also.